src/client/game.cpp

```diff
@@ -335,6 +354,29 @@ class GameGlobalShaderConstantSetter : public IShaderConstantSetter
 
 		float saturation = lighting.saturation;
 		m_saturation_pixel.set(&saturation, services);
+		video::SColorf artificial_light = lighting.artificial_light_color;
+		m_artificial_light.set(artificial_light, services);
+
+		float gamma = m_gamma;
+		m_gamma_pixel.set(&gamma, services);
+
+		const Vignette &vignette_params = lighting.vignette;
+		m_vignette_dark_pixel.set(&vignette_params.dark, services);
+		m_vignette_bright_pixel.set(&vignette_params.bright, services);
+		m_vignette_power_pixel.set(&vignette_params.power, services);
```

Comment:
shouldn't this be inside `if (m_bloom_enabled)`?

Reply:
No, why?

Reply:
I thought the shader only used this value with bloom enabled, must have been a mistake.